src/bindings/jna/libelektra4j/src/main/java/org/libelektra/KeySet.java

```diff
@@ -40,10 +40,14 @@ protected KeySet(final long p) {
 
 	/**
 	 * Helper constructor for duplication by pointer
+	 * The pointer may not be null or an IllegalArgumentException is thrown
 	 *
 	 * @param p Pointer to another KeySet
 	 */
-	protected KeySet(final Pointer p) {
+	public KeySet(final Pointer p) {
+		if (p == null) {
+			throw new IllegalArgumentException("Passed pointer may not be null");
```

Comment:
Should be NullPointerException?

```suggestion
			throw new IllegalArgumentException("Passed pointer should not be null");
```